server/src/rbx_snapshot.rs

```diff
@@ -202,24 +212,27 @@ pub fn snapshot_project_node<'source>(
 ) -> SnapshotResult<'source> {
     let maybe_snapshot = match &node.path {
         Some(path) => snapshot_imfs_path(context, imfs, &path, Some(instance_name.clone()))?,
-        None => match &node.class_name {
-            Some(_class_name) => Some(RbxSnapshotInstance {
+        None => {
+            let class_name = match &node.class_name {
+                Some(_class_name) => Cow::Borrowed("Folder"),
+                None => infer_class_name(instance_name.clone())
+                    .ok_or(SnapshotError::ProjectNodeUnusable)?,
+            };
```

Comment:
Does the comment listed below still hold now that we set `class_name` to something besides `Folder`?

https://github.com/rojo-rbx/rojo/pull/210/files#diff-6dbbd20807104f15267e0279a30e80beR259